petric.py

```diff
@@ -165,7 +198,18 @@ def get_data(srcdir=".", outdir=OUTDIR, sirf_verbosity=0):
         penalty_strength = 1 / 700 # default choice
     prior = construct_RDP(penalty_strength, OSEM_image, kappa)
 
-    return Dataset(acquired_data, additive_term, mult_factors, OSEM_image, prior, kappa)
+    reference_image = STIR.ImageData(str(srcdir / 'reference_image.hv')) if (srcdir /
+                                                                             'reference_image.hv').is_file() else None
+    background_mask = STIR.ImageData(str(srcdir / 'VOI_background.hv')) if (srcdir /
+                                                                            'VOI_background.hv').is_file() else None
+    foreground_mask = STIR.ImageData(str(srcdir / 'VOI_foreground.hv')) if (srcdir /
+                                                                            'VOI_foreground.hv').is_file() else None
+    voi_masks = {
+        voi.stem: STIR.ImageData(str(voi))
+        for voi in srcdir.glob("VOI_*.hv") if voi.stem[4:] not in ('background', 'foreground')}
```

Comment:
@KrisThielemans please check this naming convention.

btw as an alternative, I would be fine with bitmasking so e.g.

```python
voi_flags = ImageData('VOI.hv')
num_flags = int(np.log2(voi_flags.max()))
voi_masks = [np.clip(voi_flags & (1 << f), 0, 1) for f in range(num_flags + 1)]

background_mask, foreground_mask = voi_masks[:2]
voi_masks = voi_masks[2:]
```

Reply:
Let's cuahge `VOI_foreground.hv` to `VOI_whole_object.hv`